### PR TITLE
feat(Charles): add initial implementation

### PR DESCRIPTION
**Pull request by eljefedelrodeodeljefe**
Signed-off-by: Robert Jefe Lindstaedt <robert.lindstaedt@gmail.com>

---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

Note: I believe the host injection cannot be tested before deployment. At least we have found no way around it. Hence we are testing against cucstom connector

### CODE REVIEW

File: certified-connectors/Charles Universe/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Charles Universe
```
Comment (sriyen-msft) [inline certified-connectors/Charles Universe/README.md:1]: @eljefedelrodeodeljefe Please provide a detailed description of your connector which anyone can understand.

File: certified-connectors/Charles Universe/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,232 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Charles Universe",
+    "description": "The Charles Universe connector allows to react to events your Universe installation is generating. \n\n That is a e.g. send a Teams or Slack message when a person is created.",
```
Comment (sriyen-msft) [inline certified-connectors/Charles Universe/apiDefinition.swagger.json:5]: @eljefedelrodeodeljefe The description can be improved a bit, Can you please make it free from grammatical mistakes. Ex.: `...This is an e.g. send...`

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@eljefedelrodeodeljefe I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have.

File: certified-connectors/Charles Universe/README.md (hunk 1 above)
Comment (ptalrejagithub) [reply certified-connectors/Charles Universe/README.md:1]: @eljefedelrodeodeljefe , you can refer to the readme.md [here](https://github.com/st0014/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md). readme.md file is only required for open sourcing your connector which can be helpful to the open source users who may want to contribute in your connector.

**Comment by sriyen-msft** [comment]
@eljefedelrodeodeljefe I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have.

**Comment by eljefedelrodeodeljefe** [comment]
@sriyen-msft @ptalrejagithub sorry for the delay. I believe we have addressed everything and hope this is compliant with your guidelines.

Thank your for your review.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**